barretenberg/cpp/src/CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ include(GNUInstallDirs)
 message(STATUS "Compiling all-in-one barretenberg archive")
 
 set(BARRETENBERG_TARGET_OBJECTS
-    $<TARGET_OBJECTS:client_ivc_objects>
+    $<TARGET_OBJECTS:aztec_ivc_objects>
```

Comment:
I was confused why this would build bc bb uses ClientIVC and not AztecIVC. I think the answer is that it gets both already through dsl, so maybe you could remove this and it would still build?

Reply:
Oh its just that client_ivc_objects was accidentally duplicated in this file so I just changed one of the duplicates to aztec. Now theyre both included